wpengine/atlas-blueprint-portfolio
# Merl 739 remove duplicate blog and portfolio from menu

Description: This PR fixes the menu duplications in atlas-blueprint-portfolio when launched from Local or the Atlas portal.

To Test: 
1. Go to Local and Create a new site. 
![image](https://user-images.githubusercontent.com/50935135/218192773-0687e497-0242-48b3-98a3-66b405f3b907.png)

Select `Custom,` set your PHP, Web server, and Database options, and then select 

- [ ] `enable Atlas Add-on on this site.`

![image](https://user-images.githubusercontent.com/50935135/218192892-bfac2bb5-342e-4ebd-9179-da3fc314abb0.png)

2. Open up your site folder in VScode.
![image](https://user-images.githubusercontent.com/50935135/218193405-1a58464a-a0b0-45d5-aa0b-b0362dcbd382.png)

Open the `app-node` folder, and copy your `env.local` file. Set aside.
Delete the `app-node` folder.

From your terminal, run:
`git clone https://github.com/wpengine/atlas-blueprint-portfolio.git app-node`
Place your`env.local` file in your freshly generated `app-node` folder.
Change your directory to `app-node` and run `npm install`

3. Now go back to Local, head to WP Admin, and add the ACM plugin.

4. Once you have done so, open up your site's shell from Local and run these commands:
`wp acm reset --yes --all` (to clean the site of any sample pages or posts that are automatically installed).
`wp acm blueprint import https://github.com/wpengine/atlas-blueprint-portfolio/raw/MERL-739-remove-duplicate-blog-and-portfolio-from-menu/acm-blueprint.zip
` (to import this PR's new acm-blueprint.zip file)

5. Go to Local and `Open site`. Check the layout change. Note whether the menu items are duplicated. (**Note**: you may need to close Local entirely and restart if you are not seeing the change take effect).

![image](https://user-images.githubusercontent.com/50935135/218194709-8dde3a83-744f-460c-b51b-f7001616883b.png)
